controllers/controllers.js

```diff
@@ -30,8 +30,16 @@ const getArticleById = (req, res, next) => {
     })
 }
 
+const getArticles = (req, res, next) => {
+    const orderBy = 'created_at'
+    console.log('controller 1')
 
+    selectAllArticles(orderBy).then(data => {
```

Comment:
'data' could be more descriptive.  could use articles.  then you can do  .send({articles}) instead of .send({ articles: data })